samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -203,6 +218,28 @@ export class FormComponent implements OnInit {
    */
   handleStep(step?: FRStep) {
     this.step = step;
+
+    /**
+     * If the step contains TextOutputCallback, it means it has webAuthn callbacks so the sign-in button must be hidden as
+     * webAuthn will provide both biometric registration and biometric authentication log-in option buttons.
```

Comment:
This comment is technically correct, and I don't want our customers to be misled by it. If a step has a `TextOutputCallback`, it doesn't mean it's WebAuthn. Our WebAuthn node does use `TextOutputCallback`, but it could also use `MetadataCallback`. Because of this, the presence of this callback doesn't indicate anything. The only way we can know this is a WebAuthn step is by using the dedicated `isWebAuthnType` method.

Reply:
got it, shall i remove it then?

Reply:
Ugh, sorry. I have a typo in my original comment. I meant to write, "This comment is NOT technically correct ... " Also, I'm not talking directly about the comment itself, but also the logic we are using below. Since we have access to `step` and already pull in `FRWebAuthn`, I'd rather use use the `FRWebAuthn.getWebAuthnStepType` method to derive states of UI, rather than the presence of `TextOutputCallback`. The presence of `TextOutputCallback` is not a good indicator of much, unfortunately.

Reply:
done